### DISCUSSION TITLE
Proposal: An update to the Field Guide

### DISCUSSION THREAD

**Discussion by aaronjorbin** [WordPress/developer-blog-content#169] [Topic Ideas]
Based on [this comment](https://make.wordpress.org/core/2023/08/31/proposal-an-update-to-the-field-guide/#comment-45566), I am opening this in order to discuss the proposal at https://make.wordpress.org/core/2023/08/31/proposal-an-update-to-the-field-guide/

Please leave comments on the make post.

**Comment by bph** [comment]
Thank you @aaronjorbin for putting the proposal on our radar here. It's a very interesting cross-team discussion and I hope we have more comments on the [Make post ](https://make.wordpress.org/core/2023/08/31/proposal-an-update-to-the-field-guide/)
